install/linux/usr/share/odemis/hwtest/xt-only.odm.yaml

```diff
@@ -0,0 +1,48 @@
+SEM: {
+  class: Microscope,
+  role: sem,
+  children: ["SEM EXT", "EBeam Scanner", "EBeam Focus", "EBeam Stage",
```

Comment:
Actually, the "SEM EXT" doesn't need to be a child of the sem, as it plays no role (directly).